account_banking_sepa_credit_transfer/README.rst

```diff
@@ -0,0 +1,95 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+====================================
+Account Banking SEPA Credit Transfer
+====================================
+
+Module to export payment orders in SEPA XML file format.
+
+SEPA PAIN (PAyment INitiation) is the new european standard for
+Customer-to-Bank payment instructions. This module implements SEPA Credit
+Transfer (SCT), more specifically PAIN versions 001.001.02, 001.001.03,
+001.001.04 and 001.001.05. It is part of the ISO 20022 standard, available on
+http://www.iso20022.org.
+
+The Implementation Guidelines for SEPA Credit Transfer published by the
+European Payments Council (http://http://www.europeanpaymentscouncil.eu) use
```

Comment:
The link in this line is not working. Maybe this? https://www.europeanpaymentscouncil.eu/